src/components/Visualizations/Graph/MMRGraph.jsx

```diff
@@ -18,8 +18,8 @@ const StyledGraphArea = styled.div`
 user-select: none;
 `;
 
-const filterNullValues = function (columns) {
-  const c = columns;
+const filterZeroValues = function (column) {
+  const c = column;
```

Comment:
suggest `return {...column, solo_competitive_rank: c.solo_competitive_rank || null...`

Reply:
don't need the const c = column